src/index.js

```diff
@@ -0,0 +1,94 @@
+temp = 70
+
+const updateTemp = temp => {
+    const tempFarenheitContainer= document.getElementById("tempFarenheit");
+    tempFarenheitContainer.textContent = temp;
+}
+const increaseTemp = () => {
+    temp += 1;
+    updateTemp(temp);
+};
+const decreaseTemp = () => {
+    temp -=1;
+    updateTemp(temp);
+};
+const updateTempColor = (updatedTemp)=>{
+    const tempFarenheitContainer= document.getElementById("tempFarenheit");
+    let tempColor = "black";
+    if (updatedTemp >= 80){
+        tempColor = "red"
+    }   else if (updatedTemp >=70){
+            tempColor = "orange"
+    }   else if (updatedTemp >=60){
+            tempColor = "yellow"
+    }   else if (updatedTemp >=50){
+            tempColor = "green"
+    }   else {
+            tempColor = "blue"
+    }
+    tempFarenheitContainer.textContent = tempFarenheit
+};
+
+const updateGround =(updatedTemp) =>{
+    const outsideContainer= document.getElementById("ground");
+    let groundEmojis = "🌵🏜🐍🦂🏜🌵🦂🌵🐍🐍🏜🦂";
+    if (updatedTemp <= 50){
+        groundEmojis = "🌲🌲⛄️🌲⛄️🍂🌲🌲🌲⛄️🍂🌲";
+    }   else if (updatedTemp <=70){
+            groundEmojis = "🌾🌾🛤🍃🪨🛤🍁🌾🌾🍁🍃🍁";
+    }    else if (updatedTemp <=80){
+            groundEmojis = "🌸🌿🌼🌸🌷🌻🌿🌱🌱🌼🌻🌷";
+    }
+    outsideContainer.textContent = groundEmojis
+};
+
+const updateSky =() =>{
+    const selectedSky = document.getElementById("skyOptions");
```

Comment:
This retrieves the select element itself, rather than the value it currently has. As a result, none of the following checks will be found to match (comparing a string to an HTML element). This can be addressed here by making sure that we get the `.value`.

```js
    const selectedSky = document.getElementById("skyOptions").value;
```